src/lib/agent.c

```diff
@@ -61,7 +61,9 @@ static enum task_state validate(void *data)
 
 	/* panic timeout */
 	if (sa->panic_on_delay && delta > sa->panic_timeout)
+#if CONFIG_AGENT_TIME_PANIC
 		panic(SOF_IPC_PANIC_IDLE);
+#endif
```

Comment:
This is a bug in the making. If CONFIG_AGENT_TIME_PANIC is off, you'll nest the next if...

Reply:
Oh, oopsie